android/uhabits-android/src/main/java/org/isoron/uhabits/activities/AndroidThemeSwitcher.kt

```diff
@@ -19,15 +19,16 @@
 
 package org.isoron.uhabits.activities
 
-import android.app.*
-import android.content.res.Configuration.*
-import android.os.Build.VERSION.*
-import androidx.core.content.*
-import org.isoron.androidbase.activities.*
-import org.isoron.uhabits.*
-import org.isoron.uhabits.core.preferences.*
-import org.isoron.uhabits.core.ui.*
-import javax.inject.*
+import android.app.Activity
+import android.content.Context
+import android.content.res.Configuration.UI_MODE_NIGHT_MASK
+import android.content.res.Configuration.UI_MODE_NIGHT_YES
+import android.os.Build.VERSION.SDK_INT
+import androidx.core.content.ContextCompat
+import org.isoron.androidbase.activities.ActivityScope
+import org.isoron.uhabits.R
+import org.isoron.uhabits.core.preferences.Preferences
+import org.isoron.uhabits.core.ui.ThemeSwitcher
```

Comment:
Hi @iSoron, I'm no Java/Kotlin developer myself and have nothing to do with this pull request. But I have a question, just out of curiosity: why is it that you prefer wildcard imports? Is it just a personal preference (which would be fine, mind you) or is there more reason for it? To me (again: no Java dev here) it looks like using wildcard imports would import more than what might be needed, resulting in a larger binary. But I might be terribly wrong here. 😅

On a sidenote: as this project apparently draws the attention of quite a few contributors (which is great! 🥳), maybe it would be a good idea to add a [CONTRIBUTING.md](https://docs.github.com/en/free-pro-team@latest/github/building-a-strong-community/setting-guidelines-for-repository-contributors). This could probably save you some time. 😊

Reply:
@mh166 The main reason for using wildcard imports here is to keep the code consistent with the rest of our code case. Both ways are valid and lead to exactly bytecode. Our production APK also goes through proguard, which removes not only unused classes, but also unused methods and attributes. For new code bases, I prefer wildcard imports because they work just fine and are much less verbose.

Thanks for the CONTRIBUTING.md suggestion. We already have some information in the wiki section, but it could be improved and should probably be moved to a file, which is easier to find.

Reply:
Thank you for the feedbacks. However, we are still new to android programming and we are not sure to understand well what are wildcard imports. I tried to change the import in the latest commits. Please tell me if the imports are correct.